docs/responsibilities-and-powers.md

```diff
@@ -3,3 +3,22 @@ title: Responsibilities and powers
 ---
 
 Pickup admins are responsible for the game balance, healthy organization and - above all - creating the friendly and easy atmosphere.
+
+They can:
+
+- assign skills to players, see skills of all players,
+- request player substitutes during the games,
+- ban players and revoke those bans,
+- define maps available on site,
+- define what whitelist is used on pickup games,
+- force create player accounts,
+- scramble maps available in the votes,
+- edit rules page.
```

Comment:
Technical:
> - edit static pages

I know the only static page currently available is the rules one, but soon there will be more.